packages/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -1853,8 +1857,8 @@ export class ApiModelGenerator {
 						?.map((param) => ` * @param ${param.name} - ${this._fixLinkTags(param.description) ?? ''}\n`)
 						.join('') ?? ''
 				}${
-					method.returns?.length && !Array.isArray(method.returns[0])
-						? ` * @returns ${this._fixLinkTags(method.returns[0]!.description) ?? ''}\n`
+					method.returns?.length && !Array.isArray(method.returns[0]) && method.returns[0]?.description
+						? ` * @returns ${this._fixLinkTags(method.returns[0].description) ?? ''}`
```

Comment:
This is already covered by the length check. Alternatively, you could get rid of the length check... or just leave it as it is
```suggestion
					method.returns?.length && !Array.isArray(method.returns[0]) && method.returns[0]!.description
						? ` * @returns ${this._fixLinkTags(method.returns[0]!.description) ?? ''}`
```